app/models/chef.rb

```diff
@@ -1,4 +1,8 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+
+   def chef_ingredients
+     dishes.joins(:ingredients).distinct.pluck("ingredients.name")
+   end
```

Comment:
This is great, one thing you could have done was to also add a second association the your model and therefore avoid having to do the join yourself.

```ruby
class Chef < ApplicationRecord
  validates_presence_of :name
  has_many :dishes
  has_many :ingredients, through: :dishes

  def chef_ingredients
    ingredients.distinct.pluck("ingredients.name")
  end
end
```